aspnetcore/common/samples/WebApplication1/Startup.cs

```diff
@@ -75,7 +75,29 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseIdentity();
 
-            // Add external authentication middleware below. To configure them please see http://go.microsoft.com/fwlink/?LinkID=532715
+            app.UseFacebookAuthentication(new FacebookOptions()
```

Comment:
I think the `Startup` sample is designed to be pretty basic and the link to the docs is sufficient. If you feel the sample should have a concrete example of external login, maybe keep one and provide the link to the docs for the rest?